perllib/FixMyStreet/App/Controller/Moderate.pm

```diff
@@ -299,6 +306,31 @@ sub moderate_category : Private {
     }
 }
 
+sub moderate_state : Private {
+    my ($self, $c) = @_;
+
+    my $new_state = $c->get_param('state');
+    return unless $new_state;
+
+    my $problem = $c->stash->{problem};
+    if ($problem->state ne $new_state) {
+        $problem->state($new_state);
+        $problem->update;
+        $problem->add_to_comments( {
+            text => $c->stash->{moderation_reason},
```

Comment:
Normally this doesn't get made public so is that not going to be a bit unexpected?

Reply:
Hmm, so this won't be on .com, only on a cobrand that's deliberately added state to the moderation form (like kiitc) where it's clearer it's a public editing, but yes, I guess it still might be odd. Guess I'll add a comment here at least about thinking about that for any other cobrand that might use it.